swagger.yml

```diff
@@ -2319,7 +2325,7 @@ paths:
           schema:
             $ref: "#/definitions/error"
 
-  /repositories/{repository}/actions/runs/{run_id}/hooks/{hook_id}/output:
+  /repositories/{repository}/actions/runs/{run_id}/hooks/{hook_run_id}/output:
```

Comment:
Providing another nested generated ID is less convenient for the user. Isn't `hook_run_id` sufficient for us to fetch the relevant data?